packages/avatar-group/theme/material/vaadin-avatar-group-styles.js

```diff
@@ -2,6 +2,48 @@ import '@vaadin/vaadin-material-styles/color.js';
 import { menuOverlay } from '@vaadin/vaadin-material-styles/mixins/menu-overlay.js';
 import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
+registerStyles(
+  'vaadin-avatar-group',
+  css`
+    :host {
+      --vaadin-avatar-size: 36px;
+    }
+
+    :host([theme~='small']) {
```

Comment:
Let's extract this change (adding theme variants) into a separate PR, as it's essentially a new feature, not directly related to the original issue.

Reply:
I have extracted this part to another pull request: https://github.com/vaadin/web-components/pull/4106